### ISSUE TITLE
Add exception for Immersive Fox class .share-icon (False Positive of Fanboy’s Annoyance)

### ISSUE THREAD

**Issue by lcsvcn** [uBlockOrigin/uAssets#15686]
### Prerequisites

- [X] This is not a [support issue or a question](https://old.reddit.com/r/uBlockOrigin/)
- [X] I performed a [cursory search of the issue tracker](https://github.com/uBlockOrigin/uAssets/issues) to avoid opening a duplicate issue
- [X] I am running the latest version of uBlock Origin

### I tried to reproduce the issue when...

- [X] uBO is the only extension
- [X] uBO with default lists/settings
- [X] using a new, unmodified browser profile

### URL(s) where the issue occurs

`https://dev.immersive-fox.com/saved`,
`https://immersive-fox.com/saved`

### Describe the issue

I work at Immersive Fox (http://immersive-fox.com/ and http://dev.immersive-fox.com/) and some filters are blocking a core functionality of our website. We do not have any social annoyances at all, but we do have a share icon (with .share-icon to their css) to allow users to share they generated video.

There was a related issue at Adguard filters (not the same issue, but I was wrongly hiding the same element), but they updated now to avoid removing share icon, since is not any annoyance, but a core feature of our paid service. If you want to take a look here is the issue, here is a link:

https://github.com/AdguardTeam/AdguardFilters/issues/134966

### Screenshot(s)

<details><summary>Social Fanboy’s Annoyance Off:</summary>

<img width="274" alt="Screenshot 2022-11-15 at 07 50 43" src="https://user-images.githubusercontent.com/6011385/201905388-b4ff5727-b692-4c72-9ba3-0682e37bc562.png">
</details>
<details><summary>Social Fanboy’s Annoyance On:</summary>
<img width="277" alt="Screenshot 2022-11-15 at 07 51 11" src="https://user-images.githubusercontent.com/6011385/201905356-a318d704-27c6-4b7c-81b7-bb3dbb3e9ccd.png">
</details>

<details><summary>Icon Html Code:</summary>
<img width="250" alt="Screenshot 2022-11-15 at 07 51
<img width="858" alt="Screenshot 2022-11-15 at 07 50 35" src="https://user-images.githubusercontent.com/6011385/201905406-f531f65d-2578-4f05-add6-a19b6b49af52.png">
</details>

<details><summary>Filter Logs:</summary>
<img width="1431" alt="Screenshot 2022-11-16 at 08 30 39" src="https://user-images.githubusercontent.com/6011385/202169345-6461eed4-5436-4555-9800-9efc9c82042d.png">
<img width="1414" alt="Screenshot 2022-11-16 at 08 30 28" src="https://user-images.githubusercontent.com/6011385/202169366-770431c9-d6fc-4031-a309-8d839642c06f.png">

</details>

### uBlock Origin version

1.45.2

### Browser name and version

Firefox 106.0.5

### Settings

Here is my ublock setup
[my-ublock-backup_2022-11-16_12.13.11.txt](https://github.com/uBlockOrigin/uAssets/files/10021151/my-ublock-backup_2022-11-16_12.13.11.txt)

### Notes

I have noticed that this issue happens with updated Firefox version and do not happen with Chrome updated version of Ublock.

**Comment by lcsvcn** [comment]
PR to fix this issue on Easylist: https://github.com/easylist/easylist/pull/14137

**Comment by gwarser** [comment]
Isn't this already fixed yesterday https://github.com/easylist/easylist/commit/a6a19fa01cc134e2c3894d5a7f08cc3dffdd0729 ?

**Comment by mapx-** [comment]
You have to update fanboy's annoyances list / social list

**Comment by lcsvcn** [comment]
Yeah, I did that three times already. It is still blocking .share-icon

**Comment by mapx-** [comment]
I see `Version: 202211120040` maybe it's server issue

**Comment by krystian3w** [comment]
https://github.com/uBlockOrigin/uBlock-issues/discussions/2340#discussioncomment-3971701

**Comment by mapx-** [comment]
Now I got the 17.11 version .. cloudflare issue .. @lcsvcn try & try again

**Comment by lcsvcn** [comment]
Yeah, now it worked for me as well. Thank you! I got the updated version.
